### ISSUE TITLE
Implement setup for multi-label classification

### ISSUE THREAD

**Issue by comsaint** [comsaint/dlh_project#17]
Previously in @AsadBinImtiaz 's branch, the `Dataset` object generates multiple labels (e.g. [1, 0, 0, 0, 1, 1, ..., 0]) for each image, which is good for multi-label classification. Unfortunately, I overwrote the work due to time constraint but implemented setting for binary classification instead. Now it is a good time to implement the multi-label setting.

Tasks:
1. Edit `data_processing.py` and `dataset.py` so that the data loader produces a label vector (instead of 1 label).
2. Change the loss function to binary cross-entropy loss.
3. Modify the training and evaluation functions in `train_model.py` to adapt for new loss and metric (Macro-average ROC?). Also, pay attention to the last classification layer, making sure it is **not** using Softmax.
4. (Question) how to handle class weight and stratify strategy?
5. (Question) how to implement early stopping?

Notes:
1. Do not confuse multi-class with multi-label classification. [link](https://towardsdatascience.com/multi-label-image-classification-with-neural-network-keras-ddc1ab1afede)
2. Good reference on loss function. [link](https://gombru.github.io/2018/05/23/cross_entropy_loss/)
